### PR TITLE
Zhaofeng/update readme 

### PR DESCRIPTION
**Pull request by xzf0587**
1. improve SQL help in readme.

### CODE REVIEW

File: share-now/README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Share Now promotes the exchange of information between colleagues by enabling us
 1. Open the project in Visual Studio Code.
 1. Open the command palette and select: `Teams: Provision in the Cloud`. You will be asked to input admin name and password of SQL. The toolkit will help you to provision Azure SQL.
 1. Once provision is completed, open the command palette and select: `Teams: Deploy to the Cloud`.
-1. Open [.fx/env.default.json](.fx/env.default.json) file, you could get the database name in `databaseName` setting. In Azure portal, find the database and use [query editor](https://docs.microsoft.com/en-us/azure/azure-sql/database/connect-query-portal) with below query to create tables:
+1. Open [.fx/env.default.json](.fx/env.default.json) file, you could get the sql endpoint in `sqlEndpoint` setting and database name in `databaseName` setting. [Add IP address of your computer into allowlist of firewall of Azure SQL Database](https://docs.microsoft.com/en-us/azure/azure-sql/database/firewall-configure#from-the-database-overview-page). In Azure portal, find the database and use [query editor](https://docs.microsoft.com/en-us/azure/azure-sql/database/connect-query-portal) with below query to create tables:
```
Comment (adashen) [inline share-now/README.md:32]: shall we separate it as two steps?

File: todo-list-with-Azure-backend/README.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ Todo List provides an easy way to manage to-do items in Teams Client. This app h
 1. Open the project in Visual Studio Code.
 1. Open the command palette and select `Teams: Provision in the Cloud`. You will be asked to input admin name and password of SQL. The toolkit will help you to provision Azure SQL.
 1. Once provision is completed, open the command palette and select `Teams: Deploy to the Cloud`.
-1. Open [.fx/env.default.json](.fx/env.default.json) file, you could get the database name in `databaseName` setting. In Azure portal, find the database and use [query editor](https://docs.microsoft.com/en-us/azure/azure-sql/database/connect-query-portal) with below query to create a table:
+1. Open [.fx/env.default.json](.fx/env.default.json) file, you could get the sql endpoint in `sqlEndpoint` setting and database name in `databaseName` setting. [Add IP address of your computer into allowlist of firewall of Azure SQL Database](https://docs.microsoft.com/en-us/azure/azure-sql/database/firewall-configure#from-the-database-overview-page). In Azure portal, find the database and use [query editor](https://docs.microsoft.com/en-us/azure/azure-sql/database/connect-query-portal) with below query to create a table:
```
[the remark this answers is not in the document]
Comment (xzf0587) [reply todo-list-with-Azure-backend/README.md]: It is fixed. Thanks.

File: share-now/README.md [hunk 3]
```diff
@@ -29,7 +29,8 @@ Share Now promotes the exchange of information between colleagues by enabling us
 1. Open the project in Visual Studio Code.
 1. Open the command palette and select: `Teams: Provision in the Cloud`. You will be asked to input admin name and password of SQL. The toolkit will help you to provision Azure SQL.
 1. Once provision is completed, open the command palette and select: `Teams: Deploy to the Cloud`.
-1. Open [.fx/env.default.json](.fx/env.default.json) file, you could get the database name in `databaseName` setting. In Azure portal, find the database and use [query editor](https://docs.microsoft.com/en-us/azure/azure-sql/database/connect-query-portal) with below query to create tables:
+1. Open [.fx/env.default.json](.fx/env.default.json) file, you could get the SQL endpoint in `sqlEndpoint` setting and database name in `databaseName` setting. [Add IP address of your computer into allowlist of firewall of Azure SQL Database](https://docs.microsoft.com/en-us/azure/azure-sql/database/firewall-configure#from-the-database-overview-page).
```
Comment (adashen) [inline share-now/README.md:32]: firewall setting is in server level right not database?

Comment (xzf0587) [reply share-now/README.md]: It is fixed. Thanks.

**Pull request merged**